checklists/apim_checklist.en.json

```diff
@@ -144,13 +164,53 @@
         {
             "category": "Management",
             "subcategory": "Business continuity and disaster recovery",
-            "text": "Deploy at least two scale units spread over two availability zones per region for best availability and performance",
+            "text": "Leverage multi-region deployment of API gateways (available only in premium tiers). APIM routes the requests to a regional gateway based on the lowest latency. You could use your own Traffic Manager with custom routing rules.",
```

Comment:
I get the feeling this is very similar to previous, both recommending multi-region deployments. Wouldn't it better to stick to one?

Reply:
This is a duplicate.

Reply:
Removed the one with ATM, probably too many details